aiohttp/errors.py

```diff
@@ -44,6 +46,23 @@ def message(self):
         return self.args[1]
 
 
+class WSServerDisconnectedError(ClientDisconnectedError):
```

Comment:
I think parent should be `ServerDisconnectedError`, not `ClientDisconnectedError`.